user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Other/Grid.md

```diff
@@ -14,6 +14,8 @@ With this component, you can:
 
 - Instantly assess the status of items, thanks to [conditional formatting](#customizing-grid-items), e.g. red may indicate a service is in alarm.
 
+- [Select a grid item](#using-the-grid-as-a-selector) to filter or update the content shown in other parts of your dashboard or app.
```

Comment:
The most complete way to describe this would be that the grid selections provides (AKA feeds) data to other components. It's necessarily updating or showing data, could also be some argument of scripts that change. The functionality of selecting rows in a table is actually identical.